cli/gardener_ci/_oci.py

```diff
@@ -24,12 +24,16 @@ def ls(image: str):
     print(oci_client.tags(image_reference=image))
 
 
-def manifest(image_reference: str):
+def manifest(image_reference: str, pretty:bool=True):
```

Comment:
nit: inconsistent within the line)
```suggestion
def manifest(image_reference: str, pretty: bool=True):
```